sympy/integrals/intpoly.py

```diff
@@ -0,0 +1,796 @@
+from __future__ import print_function, division
+
+from functools import cmp_to_key
+
+from sympy.core import S, Symbol
+
+from sympy.geometry import Segment2D
+from sympy.geometry.polygon import Polygon
+from sympy.geometry.point import Point
+from sympy.core.expr import Expr
+from sympy.abc import x, y
+from sympy.core.function import diff
+
+from sympy.polys.polytools import LC
+from sympy.polys.polytools import gcd_list
+
+
+def polytope_integrate(poly, expr, **kwargs):
+    """Pre-processes the input data for integrating
+    univariate/bivariate polynomials over 2-Polytopes.
```

Comment:
Isn't this the main function in this module that the user will call?

In that case, let's change it to something like:
```
"""
Integrates homogeneous functions over polytopes.

This function accepts the polytope in `poly` (currently only polygons are implemented) and the homogeneous function in `expr` (currently only univariate/bivariate polynomials are implemented) and returns the exact integral of `expr` over `poly`.

Parameters
...
```

Essentially imagine a user of sympy who has never heard of your module, or your function and let's say the user loads the documentation in a jupyter notebook using `polytope_integrate?`. Then just the docstring should provide enough information to understand what a general idea what this is doing, and then go into more and more details.